File: frameworks/Java/restexpress/src/main/java/hello/controller/PlaintextController.java
package hello.controller;

import org.restexpress.Request;
import org.restexpress.Response;

public class PlaintextController
{
	private static final String MESSAGE = "Hello, World!";

	public String sayHello(Request request, Response response)
	{
		response.setContentType("text/plain");
		return MESSAGE;
	}
}

File: frameworks/PHP/symfony/src/Controller/PlaintextController.php
<?php

namespace App\Controller;

use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class PlaintextController
{
    /**
     * @Route("/plaintext")
     */
    public function plaintext(): Response
    {
        return new Response('Hello, World!', 200, ['Content-Type' => 'text/plain']);
    }
}
